### PR TITLE
data:coder-chahal

### PR DESCRIPTION
**Pull request by coder-chahal**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5336"></a>

### CODE REVIEW

File: data/coder-chahal.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Sara Jaoude",
```
Comment (ChinmayMhatre) [inline data/coder-chahal.json:2]: I believe you missed this, update this with your name

File: data/coder-chahal.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "type": "personal",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
```
Comment (ChinmayMhatre) [inline data/coder-chahal.json:4]: Please update the bio with your own

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Mentioned some changes for things that you might have missed . overall, looks good! Thank you!

File: data/coder-chahal.json (hunk 1 above)
Comment (priyankarpal) [inline data/coder-chahal.json:2]: hi @coder-chahal please remove `Sara Jaoude` and add your name. 
``` suggestion
  "name": "coder-chahal",
```

File: data/coder-chahal.json (hunk 2 above)
Comment (priyankarpal) [inline data/coder-chahal.json:4]: Also please add your bio.

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/coder-chahal.json (hunk 2 above)
Comment (coder-chahal) [reply data/coder-chahal.json]: @ChinmayMhatre Updated Successfully.

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Welcome to LinkFree. 🎉

**Pull request merged**